### DISCUSSION TITLE
License doubt

### DISCUSSION THREAD

**Discussion by TheSTL** [codesandbox/sandpack#257] [Q&A] [answered]
Can I use Sandpack in a closed-source project for commercial use without modifications in the sandpack? If using it for a commercial purpose requires to open-source our code base?

**Comment by tromika** [comment, marked as answer]
Hey @TheSTL, thanks for raising this question! Currently, Sandpack has a GNU GPL license but we're working on changing it to a more permissive license. Stay tuned!

**Reply by tromika** [reply]
From today it's licensed under Apache 2.0 so this won't be a problem anymore https://twitter.com/codesandbox/status/1486330144637730817

**Reply by TheSTL** [reply]
Thanks @tromika
